File: HolaMundoJava/app/src/main/java/com/example/holamundojava/backend/Operaciones.java
package com.example.holamundojava.backend;

public class Operaciones {
    private int x;
    private int y;

    public Operaciones(int x, int y) {
        this.x = x;
        this.y = y;
    }

    public int getX() {
        return x;
    }

    public void setX(int x) {
        this.x = x;
    }

    public int getY() {
        return y;
    }

    public void setY(int y) {
        this.y = y;
    }

    public int suma(){
        return this.x + this.y;
    }
    public int resta(){
        return this.x - this.y;
    }
    public int multiplicacion(){
        return this.x * this.y;
    }
    public String division(){
        if(this.y == 0)
            return "No se puede dividir por cero";
        else
            return String.valueOf((double)this.x / this.y);
    }
}

File: HolaMundoKotlin/app/src/main/java/com/example/holamundokotlin/backend/Operaciones.kt
package com.example.holamundokotlin.backend

class Operaciones(private var x: Double, private var y: Double) {
    fun suma():Double{
        return this.x + this.y
    }
    fun resta(): Double{
        return this.x - this.y
    }
    fun multiplicacion():Double{
        return this.x * this.y
    }
    fun division(): String{
        if(this.y!=0.0)
            return (this.x / this.y).toString()
        else
            return "No se puede dividir por cero"
    }
    fun setX(x:Double){
        this.x=x
    }
    fun getX():Double{
        return this.x
    }
    fun setY(y:Double){
        this.y=y
    }
    fun getY():String{
        return (this.y).toString()
    }
}